charts/data-dashboard-backend/values.yaml

```diff
@@ -0,0 +1,118 @@
+# Default values for data-dashboard-backend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# -- Number of replicas to deploy
+replicaCount: 2
+
+image:
+  # -- docker image repository
+  repository: ghcr.io/thehyve/radar-data-dashboard-backend
+  # -- image pull policy
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: v0.1.3-alpha
```

Comment:
Since we added an option to read this from appVersion I suggest removing this to avoid duplication

Reply:
Done.